### PR TITLE
docs: adding documentation for runner-ui cli flags

### PR DESCRIPTION
**Pull request by mschile**

### CODE REVIEW

File: docs/api/commands/screenshot.mdx [hunk 1]
```diff
@@ -44,19 +44,19 @@ for more.
...
+| `capture`                    | `'fullPage'`                                                   | Which parts of the Cypress Test Runner to capture. This value is ignored for element screenshot captures. Valid values are `viewport`, `fullPage`, or `runner`. When `viewport`, the application under test is captured in the current viewport. When `fullPage`, the application under test is captured in its entirety from top to bottom. When `runner`, the entire browser viewport, including the Cypress Command Log, is captured. For screenshots automatically taken on test failure, capture is always coerced to `runner`. When [Test Replay](TODO: add link to test replay) is enabled and the Runner UI is hidden, a `runner` screenshot will not include the Runner UI and will instead capture the application under test in the current viewport. |
```
Comment (jennifer-shehane) [inline docs/api/commands/screenshot.mdx:51]:
```suggestion
| `capture`                    | `'fullPage'`                                                   | Which parts of the Cypress Test Runner to capture. This value is ignored for element screenshot captures. Valid values are `viewport`, `fullPage`, or `runner`. When `viewport`, the application under test is captured in the current viewport. When `fullPage`, the application under test is captured in its entirety from top to bottom. When `runner`, the entire browser viewport, including the Cypress Command Log, is captured. For screenshots automatically taken on test failure, capture is always coerced to `runner`. When [Test Replay](TODO: add link to test replay) is enabled and the Runner UI is hidden, a `runner` screenshot will not include the Runner UI and will instead capture the application under test only in the current viewport. |
```

File: docs/guides/guides/command-line.mdx [hunk 2]
```diff
@@ -125,13 +125,15 @@ cypress run [options]
...
 | `--quiet`, `-q`                | If passed, Cypress output will not be printed to `stdout`. Only output from the configured [Mocha reporter](/guides/tooling/reporters) will print.                                            |
 | `--record`                     | [Whether to record the test run](#cypress-run-record-key-lt-record-key-gt)                                                                                                                    |
 | `--reporter`, `-r`             | [Specify a Mocha reporter](#cypress-run-reporter-lt-reporter-gt)                                                                                                                              |
 | `--reporter-options`, `-o`     | [Specify Mocha reporter options](#cypress-run-reporter-lt-reporter-gt)                                                                                                                        |
+| `--runner-ui`,                 | [Displays the Cypress Runner UI. Useful for when [Test Replay](TODO: add test replay link) is enabled and you would still like the Cypress Runner UI to be displayed](#cypress-run-runner-ui) |
```
Comment (jennifer-shehane) [inline docs/guides/guides/command-line.mdx:136]:
```suggestion
| `--runner-ui`,                 | [Displays the Cypress Runner UI. Useful for when [Test Replay](TODO: add test replay link) is enabled and you would still like the Cypress Runner UI to be displayed](#cypress-run-runner-ui) for screenshots and video |
```

File: docs/guides/guides/command-line.mdx [hunk 3]
```diff
@@ -397,6 +408,16 @@ You can specify reporter options using the
...
 
+#### `cypress run --runner-ui` {#cypress-run-runner-ui}
+
+To force Cypress to render the Runner UI, use `--runner-ui`.
+
+```shell
+cypress run --runner-ui
+```
+
+Useful for when [Test Replay](TODO: add link to test replay) is enabled and you would still like the Runner UI to be rendered.
```
Comment (jennifer-shehane) [inline docs/guides/guides/command-line.mdx:419]:
```suggestion
Useful for when [Test Replay](TODO: add link to test replay) is enabled and you would still like the Runner UI to be rendered for screenshots and video.
```

**Review by jennifer-shehane — APPROVED** [review]
Just minor changes. Looks good. Thanks Matt!

**Review by AtofStryker — APPROVED** [review]

**Pull request merged**